modules/siddhi-core/src/main/java/org/wso2/siddhi/core/util/SessionContainer.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.siddhi.core.util;
+
+import org.wso2.siddhi.core.event.SessionComplexEventChunk;
+import org.wso2.siddhi.core.event.stream.StreamEvent;
+
+import java.io.Serializable;
+
+/**
+ * This keeps the information of a session key. i.e. current session and the previous session
+ */
+public class SessionContainer implements Serializable {
+
+    private String key;
+    private SessionComplexEventChunk<StreamEvent> currentSession;
+    private SessionComplexEventChunk<StreamEvent> previousSession;
+
+    public SessionContainer(String key) {
+        currentSession = new SessionComplexEventChunk<>(key);
+        previousSession = new SessionComplexEventChunk<>(key);
+        this.key = key;
+    }
+
+    public SessionContainer() {
```

Comment:
Please remove if you don't use this constructor

Reply:
Same as the above one, it is used to create expired event chunk, and initially it does not have a session key